Add note about the limit on pro plan (#1947)
vercel/docs: pages/docs/v2/platform/limits.mdx

## Serverless Function Regions

> It is possible to deploy [Serverless Functions](/docs/v2/serverless-functions/introduction/) to multiple regions, however this featured is limited to [Enterprise plans](/pricing). See the [Picking a Region](/docs/v2/serverless-functions/introduction#picking-a-region) for how to decide a region.

Reviewer: ```suggestion
It is possible to deploy [Serverless Functions](/docs/v2/serverless-functions/introduction/) to multiple regions, however this featured is limited to [Enterprise plans](/pricing). See [Picking a Region](/docs/v2/serverless-functions/introduction#picking-a-region) to learn more about selecting a region.
```

When attempting to deploy Serverless Functions to multiple regions on a non-Enterprise plan, the deployment will fail before entering the [build step](/docs/v2/build-step/).

<Note>

>   On the Pro plan, the limitation has been around since the current pricing launch but was applied on `July 1st 2020`. For projects created after the date, it's not possible to deploy to multiple regions.

Author: The date is subject to change.
